design-at-cornell/src/components/HomepageStyles.ts

```diff
@@ -131,4 +222,87 @@ export const Grid = styled.div`
   height: fit-content;
   margin-top: 90px;
   justify-content: center;
+  @media (max-width: 1000px) {
+    display: flex;
+    flex-direction: row;
+    height: fit-content;
+    justify-content: left;
+    margin-top: 0px;
+  }
+`;
+
+export const MobileElementContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-weight: normal;
+  margin-bottom: 50px;
+
+  p {
+    font-size: 24px;
+    text-align: left;
+    align-items: left;
+    width: 300px;
+    padding-top: 25px;
+  }
+
+  img {
+    cursor: pointer;
+  }
+`;
+
+export const ListsEvents = styled.div`
+  display: flex;
+  align-items: left;
+  margin-top: 100px;
+  margin-left: 40px;
+  margin-right: 40%;
+  flex-direction: column;
+  justify-content: left;
+  height: 290px;
+`;
+
+export const ListsArticles = styled.div`
```

Comment:
Similar to the comment about MobileElementContainer, for ListArticles you can use `export const ListArticles = styled(ListEvents)` and just change the height.